docs/other-topics/upgrade.md

```diff
@@ -40,6 +40,8 @@ Head to our [Versioning Policy page](/releases) to see exactly which databases a
 
 ### Blocking access to `/lib`
 
+*Pull Request [#14352]*
```

Comment:
Not so sure if we need this, but it can't hurt to add

Reply:
For large or controversial changes, it can give more information explaining why the change was done